client/environment_test.go

```diff
@@ -5,164 +5,133 @@ import (
 	"testing"
 
 	"github.com/quintilesims/layer0/common/models"
-	"github.com/quintilesims/layer0/common/testutils"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestCreateEnvironment(t *testing.T) {
-	handler := func(w http.ResponseWriter, r *http.Request) {
-		testutils.AssertEqual(t, r.Method, "POST")
-		testutils.AssertEqual(t, r.URL.Path, "/environment/")
+	req := models.CreateEnvironmentRequest{
+		EnvironmentName:  "name",
+		InstanceSize:     "size",
+		MinClusterCount:  1,
+		UserDataTemplate: []byte("user_data"),
+		OperatingSystem:  "os",
+		AMIID:            "ami",
+	}
 
-		var req models.CreateEnvironmentRequest
-		Unmarshal(t, r, &req)
+	handler := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
[nit] would we still want to call this 'handler'?

Reply:
That matches the name of the argument we pass to `httptest.NewServer` - see https://golang.org/pkg/net/http/httptest/#NewServer

Reply:
👍